src/game/__tests__/eop.test.js

```diff
@@ -41,27 +42,40 @@ describe('game', () => {
     );
   });
 
-  it('should start with the play phase', () => {
-    expect(players["0"].getState().ctx.phase).toBe('play');
+  it('should start in play stage', () => {
+    expect(players['0'].getState().ctx.activePlayers).toBeFalsy();
   });
 
+  it('should start with no activePlayers', () => {
+    expect(players['0'].getState().ctx.activePlayers).toBeFalsy();
+  })
+
   it('should start with scores set to zero', () => {
     expect(players["0"].getState().G.scores).toStrictEqual([0,0,0]);
   });
 
-  it('should respect the play order', () => {
-    expect(players["0"].getState().G.order[0]).toBe(parseInt(players["0"].getState().ctx.currentPlayer));
+  it('have correct order in ctx', () => {
+    const state = players['0'].getState();
+    expect(state.ctx.playOrder).toStrictEqual(['0', '1', '2']);
+  });
+
+  const startingPlayer = players['0'].getState().ctx.currentPlayer;
+  
+
+  it('should not get stuck in an infinite loop', () => {
```

Comment:
That's funny. If this tests fails, it will run forever?  ;)

Reply:
I was having a problem with the tests getting stuck and I was trying to work out which lines of code were doing it. I can remove this now though because instead of failing the test will just get stuck so it's probably not very useful.

The lines are also run in the following test so if it would get stuck we'd notice it in the next test